### PR TITLE
Migrate Sentry Android to v4.3.0

### PR DESCRIPTION
**Pull request by msasikanth**
https://app.clubhouse.io/simpledotorg/story/3251/migrate-sentry-android-to-v4-3-0

### CODE REVIEW

**Comment by msasikanth** [comment]
Since this change includes Sentry upgrade please pair with @vinaysshenoy when reviewing this PR.

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -50,6 +50,18 @@
       android:name="com.google.mlkit.vision.DEPENDENCIES"
       android:value="barcode" />
 
+    <meta-data
+      android:name="io.sentry.dsn"
+      android:value="${sentryDsn}" />
+
+    <meta-data
+      android:name="io.sentry.environment"
+      android:value="${sentryEnvironment}" />
```
Comment (vinaysshenoy) [inline app/src/main/AndroidManifest.xml:59]: Huh, I didn't know this was possible.

Comment (msasikanth) [reply app/src/main/AndroidManifest.xml:59]: Yep, we can do this using `manifestPlaceHolders`: https://developer.android.com/studio/build/manifest-build-variables

**Comment by msasikanth** [comment]
**Do not merge this until the next demo release**

**Comment by vinaysshenoy** [comment]
> **Do not merge this until the next demo release**

The other option we have is to merge it and hotfix the release branch as well. This should be fine since it's an internal change.

**Comment by msasikanth** [comment]
> > **Do not merge this until the next demo release**
> 
> The other option we have is to merge it and hotfix the release branch as well. This should be fine since it's an internal change.

Okay, let's do that then 👍🏾

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (janhavisinghh) [reply app/src/main/AndroidManifest.xml:59]: @msasikanth can you explain me why it was better to add these in the manifest than the `build.gradle` file?

**Review by janhavisinghh — CHANGES_REQUESTED** [review]
LGTM. Left just one comment PTAL.

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (msasikanth) [reply app/src/main/AndroidManifest.xml:59]: WDYM? Sentry initialization after the update will be done automatically from the data provided in the manifest. PTAL at this: https://docs.sentry.io/platforms/android/

**Review by janhavisinghh — APPROVED** [review]

**Pull request merged**